stages/org.osbuild.zipl-secex.inst

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python3
+"""
+Install the Z Initial Program Loader (zipl)
+
+This will install the Z Initial Program Loader for the
+specified `kernel` with "IBM Secure Execution" enabled.
+The other required parameter are
+- `location`, which specifies is the offset of the partition containing `/se`
+within the device that is specified via `disk`.
+- `ignition_gpg_pubkey`, which specifies filename for public GPG key
+- `hostkey`, which specifies Hostkey for given IBM Z machine
+- `build_id`, which specifies CoreOS build
+"""
+
+import libarchive
+import os
+import subprocess
+import sys
+import tempfile
+
+import osbuild.api
+
+SCHEMA_2 = r"""
```

Comment:
This will need to change to an externally defined schema because of a change that just landed upstream. See https://github.com/osbuild/osbuild/pull/1663